go-controller/pkg/ovn/secondary_layer2_network_controller_test.go

```diff
@@ -335,7 +334,7 @@ func expectedLayer2EgressEntities(netInfo util.NetInfo, gwConfig util.L3GatewayC
 			StaticRoutes: []string{sr1, sr2},
 			ExternalIDs:  gwRouterExternalIDs(netInfo, gwConfig),
 			Options:      gwRouterOptions(gwConfig),
-			Policies:     []string{routerPolicyUUID1, routerPolicyUUID2},
+			Policies:     []string{routerPolicyUUID1},
```

Comment:
we should also do this for a layer 3 right? I guess we can do it as a follow up if you dont want to do it in this PR. Just let me know so we dont forget.

Reply:
Let's do that on a follow-up PR please.